server.js

```diff
@@ -8,6 +8,7 @@ var http = require('http');
 var path = require('path');
 
 app.get('/mapbox-gl.js', browserify('./js/mapbox-gl.js', {
+    ignoreTransform: ['unassertify'],
```

Comment:
We are ignoring the unassertify transform, effectively leaving the assertions in the source code as-is. Double negatives are weird.